cmd/remote-session.go

```diff
@@ -135,6 +135,7 @@ func preRunCheckEnv(c *cobra.Command, args []string) error {
 func runCreate(c *cobra.Command, args []string) error {
 	podmanargs := []string{"--remote", "run", "--rm", "-d", "--pull=always",
 		"--privileged", "--security-opt=label=disable", "--volume=/srv/",
+		"--volume=secex-data:/data.secex:ro",
```

Comment:
@dustymabe Remind me again, why can't this be a switch we pass separately to `cosa remote-session create` on just s390x? Feels odd to bake this in here.

Reply:
If i remember our discussion correctly, the main goal is to implement it in a way where we don't need to open a special case for `buildextend-secex` in the pipeline.
With having it just be there, we will be creating an empty volume on the builders once, but otherwise it does not really change anything in the workflow.

Reply:
Yeah, I see the appeal. It just feels odd to create empty volumes on other builders. I don't have a better idea. I guess in theory this code here could at least be conditional on s390x, but that would require running `ssh` (to query the architecture), which we haven't had to do so far.